charts/scalardb/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
```

Comment:
Please make it consistent with the ledger chart unless you have a reason not to do it.

For example,
Please add `terminationGracePeriodSeconds`.
https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/templates/ledger/deployment.yaml#L23

Please also add livenessProbe.

Reply:
Thanks for the suggestions. I'll check and fix them.